packages/grafana-ui/src/components/Table/RowsList.tsx

```diff
@@ -25,6 +25,8 @@ import { TableStyles } from './styles';
 import { CellColors, TableFieldOptions, TableFilterActionCallback } from './types';
 import { calculateAroundPointThreshold, getCellColors, isPointTimeValAroundTableTimeVal } from './utils';
 
+const LINE_SCALE_FACTOR = 1.12;
+const LOW_LINE_PAD = 32;
```

Comment:
I think we could move these to be defined as part of the getTextBoundingBox function as that's the only place they're being referenced